manifest.json

```diff
@@ -11,13 +11,23 @@
   },
   "applications": {
     "gecko": {
+      "id": "{09420bd6-3c35-444d-b85c-74d51fe3df4f}",
```

Comment:
Why is this necessary?

Reply:
I added this in because `browser.storage.sync` requires an extension ID. However, I don't use `browser.storage.sync`, only `browser.storage.local`, so not this can be removed if you wish.

Do you want to switch to the `storage.sync` API or continue to use `storage.local` ?

Reply:
I'd prefer `storage.local`. What do you think?

Reply:
SGTM. I have amended the commit to remove this ID.